xcode/Subconscious/Shared/Components/Common/Forms/ValidatedTextField.swift

```diff
@@ -7,17 +7,56 @@
 
 import SwiftUI
 import Combine
+import ObservableStore
+
+struct ValidatedFormField<T: Equatable, Action>: View {
```

Comment:
This wrapping view feels strange to me. Can we do it with one view?

Reply:
Yep, I was preserving the path to using the prior API but you're right, I can just flatten this out since we always use this new form.